doc/classes/Transform2D.xml

```diff
@@ -32,7 +34,7 @@
 			<param index="0" name="rotation" type="float" />
 			<param index="1" name="position" type="Vector2" />
 			<description>
-				Constructs the transform from a given angle (in radians) and position.
+				Constructs a [Transform2D] from a given rotation (in radians) and position.
```

Comment:
I'm not 100% convinced by this change. Wikipedia starts with "The radian, denoted by the symbol rad, is the unit of **angle** in the International System of Units (SI)", and *angle* is IMO more specific than *rotation*. If you tell me angle, I know more precisely what concept is meant here; it is 2d, a rotation can be 3d, if you get what I mean? Rotation is a more general concept IMO.

Reply:
I understand, that's fascinating. I spent time thinking about it and, I suppose that the float we're passing is an **angle** that is assigned to the **rotation**.

Colloquially, they mean the same thing, but yeah.